packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -4,297 +4,132 @@ import type { SliderState } from './Slider.types';
 
 export const sliderClassName = 'fui-Slider';
 
-export const thumbClassName = `${sliderClassName}-thumb`;
-export const trackClassName = `${sliderClassName}-track`;
-export const markContainerClassName = `${sliderClassName}-markItemContainer`;
-export const firstMarkClassName = `${sliderClassName}-firstMark`;
-export const lastMarkClassName = `${sliderClassName}-lastMark`;
-export const markClassName = `${sliderClassName}-mark`;
-export const markLabelClassName = `${sliderClassName}-label`;
+const thumbClassName = `${sliderClassName}-thumb`;
+const railClassName = `${sliderClassName}-rail`;
 
 /**
  * Styles for the root slot
  */
 export const useRootStyles = makeStyles({
-  root: theme => ({
+  root: {
     position: 'relative',
-    display: 'inline-flex',
+    display: 'inline-grid',
+    gridTemplateAreas: '"slider"',
     userSelect: 'none',
     touchAction: 'none',
-    verticalAlign: 'bottom',
-  }),
+  },
 
   small: {
     '--slider-thumb-size': '10px',
```

Comment:
Similar to my comment above, declare constants for these variables to avoid typos. And prefix them with `--fui-Slider`